database/README.md

```diff
@@ -0,0 +1,14 @@
+This document explains the database that the orderbook api uses.
+
+We need to store orders persistently even if the orderbook service goes. For this we use a PostgreSQL database. The schema can be found in `schema.sql`.
```

Comment:
PostgreSQL database allows us these easy resets of the data base, that we need to ensure that our event listening is robust against reorgs, right?

Reply:
The way I designed the schema it is easy to remove all events newer than some block number. This is exactly what our event based code needs.